app/Config/View.php

```diff
@@ -44,7 +44,7 @@ class View extends BaseView
      * by the core Parser by creating aliases that will be replaced with
      * any callable. Can be single or tag pair.
      *
-     * @var array<string, array<string>|callable|string>
+     * @var array<string, callable|list<string>|string>
      * @phpstan-var array<string, array<ParserCallableString>|ParserCallableString|ParserCallable>
```

Comment:
```suggestion
     * @phpstan-var array<string, list<ParserCallableString>|ParserCallableString|ParserCallable>
```

Reply:
Can you change the type names? These are not a classname, so `parser_callable` is better.
See `CodeIgniter\Config\View`.

```
 * @phpstan-type parser_callable (callable(mixed): mixed)
 * @phpstan-type parser_callable_string (callable(mixed): mixed)&string
```